README.md

```diff
@@ -2,24 +2,26 @@
 
 This repository is an *example* of how to use Databricks for setting up a robust data processing pipeline.
 
-If you are part of a Data Engineering or Data Science team, and you want to start a project in Databricks, you can use this repository as a jump-start. 
+If you are part of a Data Engineering or Data Science team, and you want to start a project in Databricks, you can use this repository as a jump start. 
 
-We tried to bake in this code some of the best practices and processes we have set up for production, research and development on top of the Databricks platform since 2017 at [Quby](https://www.quby.com/).
+Using the Databricks platform since 2017, this template shows some of the best practices and processes we have set up for production, research and development at [Quby](https://www.quby.com/).
```

Comment:
Maybe we can also remove the "since 2017" part. What do you think?

```suggestion
This template shows some of the best practices and processes we have set up for production, research and development at [Quby](https://www.quby.com/).
```